addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -37,8 +37,9 @@ class ChannelMember(models.Model):
         help="All Messages if not specified",
     )
     mute_until_dt = fields.Datetime("Mute notifications until", help="If set, the member will not receive notifications from the channel until this date.")
-    is_pinned = fields.Boolean("Is pinned on the interface", default=True)
-    last_interest_dt = fields.Datetime("Last Interest", default=fields.Datetime.now, help="Contains the date and time of the last interesting event that happened in this channel for this partner. This includes: creating, joining, pinning, and new message posted.")
+    is_pinned = fields.Boolean("Is pinned on the interface", compute="_compute_is_pinned", search="_search_is_pinned")
+    unpin_dt = fields.Datetime("Unpin date", index=True, help="Contains the date and time when the channel was unpinned by the partner.")
```

Comment:
"partner" is incorrect (can be guest as well, and really it should be user and not partner)